.github/workflows/staging-android-bte.yml

```diff
@@ -0,0 +1,82 @@
+name: Staging Android
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
similar comment here, at this point be probably only want this running on the bte branch.

Reply:
👍